src/config/database.js

```diff
@@ -30,12 +28,13 @@ class Database {
     } catch (error) {
       console.error("Unable to connect to the database:", error);
     }
-
-    Database._instance = this;
   }
 
   static getInstance() {
-    return Database._instance;
+    if (!this._instance){
+      this._instance = new Database();
+    }
+    return this._instance;
```

Comment:
not sure how this would work in theory - static methods don’t have access to this, they’re class methods. this refers to instance methods...